### DISCUSSION TITLE
VSCode Development - `code is inactive due to #[cfg] directives: kani is disabled`

### DISCUSSION THREAD

**Discussion by franklee26** [model-checking/kani#3733] [Q&A]
Hi all,

I'm interested in adding Kani to a project I'm working on. I was able to complete the installation and setup and run a basic harness from my machine. I'm using vscode w/ rust-analyzer and the Kani directives are greyed out due to `code is inactive due to #[cfg] directives: kani is disabled`. I added `"rust-analyzer.cargo.features": "all"` to my `settings.json` and restarted rust-analyzer + vscode to no avail. What is the recommended development setup to get around this?

<img width="617" alt="Screenshot 2024-11-23 at 2 20 43 PM" src="https://github.com/user-attachments/assets/12821eae-f813-4436-883a-d24eed18d9fc">

**Comment by zhassan-aws** [comment]
Can you try adding:
```
    "rust-analyzer.cargo.cfgs": {
        "kani": null,
    },
```
to `settings.json`?
